src/utils/accessibility/accessibility.js

```diff
@@ -1,22 +1,36 @@
+// 1pt = 1.333px according to http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#visual-audio-contrast-contrast-73-head
+const PIXELS_IN_POINT = 1.333;
+
 // http://www.w3.org/TR/2008/REC-WCAG20-20081211/#larger-scaledef
 function isLargeScale(fontSize, isFontBold) {
   return isFontBold ? (fontSize >= 14) : (fontSize >= 18);
```

Comment:
I got really confused here because it looked like we ignore the font size unit here.
Then, I noticed that `fontSize` actually means `fontSizeInPt` because we call `fontSizeInPt()` before calling `isLargeScale()`.
Can I suggest to add a `fontSizeUnit` argument here, and call `fontSizeInPt()` here rather than in `accessibleContrast()`? This will also ensure that if we call `isLargeScale()` from another place in the future, we don't need to remember to call `fontSizeInPt()` first.